### PR TITLE
fix(UI): Make max-width wider, and make sidebar narrower on wide screens (#1536)

### PR DESCRIPTION
**Pull request by jsit**
I think the container width isn't the primary factor in making as many posts visible on a single screen as possible.

## Before

Screenshot 2023-06-25 at 2 02 00 PM

## After

Screenshot 2023-06-25 at 2 03 25 PM

### CODE REVIEW

**Comment by SleeplessOne1917** [comment]
This is definitely a light touch that my PR. I do think the navbar would look better if it took up the full width of the screen though.

**Comment by jsit** [comment]
@SleeplessOne1917 I think the navbar contents should be limited in width, but making that shadow wrapper full-width is probably an improvement:

Screenshot 2023-06-25 at 2 40 21 PM

File: src/assets/css/main.css [hunk 1]
```diff
@@ -198,8 +198,7 @@ blockquote {
 
 .thumbnail {
   object-fit: cover;
-  min-height: 60px;
-  max-height: 80px;
+  aspect-ratio: 4/3;
```
Comment (SleeplessOne1917) [inline src/assets/css/main.css:201]: I tried this styling and the thumbnails become tiny on dektop. Is this intentional?
image

Comment (jsit) [reply src/assets/css/main.css:201]: Is this still happening? I can't seem to upload images to my local instance, so I can't see this, but this seems surprising to me.

Comment (alectrocute) [reply src/assets/css/main.css:201]: Side note – We really got to get image uploading on local instances working.

Comment (SleeplessOne1917) [reply src/assets/css/main.css:201]: @jsit  This was when I tried to apply your changes to my other PR. I'll have to test yours out locally.

> We really got to get image uploading on local instances working.

I can upload images locally when I use the dev docker container.

Comment (jsit) [reply src/assets/css/main.css:201]: @SleeplessOne1917 I can't reproduce the tiny image issue. Can you try again please?
Screenshot 2023-06-26 at 6 15 26 PM

Comment (SleeplessOne1917) [reply src/assets/css/main.css:201]: The tiny image was when I tried making some (but not the exact) changes you made in this PR to the PR I had opened earlier. This PR should be good.

File: src/shared/components/app/navbar.tsx [hunk 2]
```diff
@@ -79,256 +79,246 @@ export class Navbar extends Component<NavbarProps, NavbarState> {
...
-        <NavLink
-          id="navTitle"
-          to="/"
-          title={siteView?.site.description ?? siteView?.site.name}
-          className="d-flex align-items-center navbar-brand me-md-3"
-          onMouseUp={linkEvent(this, handleCollapseClick)}
+      <div className="shadow-sm">
+        <nav
+          className="navbar navbar-expand-md navbar-light p-0 px-3 container-lg"
+          id="navbar"
```
Comment (SleeplessOne1917) [inline src/shared/components/app/navbar.tsx:85]: I think you can accomplish the same thing without the extra div by making the nav a flexbox and setting justify content to center.

**Review by SleeplessOne1917 — APPROVED** [review]

**Comment by LizardWizardGB** [comment]
This made reading much worse, because the width of posts and comments also increased. A line of text shouldn't be no more than 90 characters long to keep readability, but now we are up at 140 characters.

**Comment by jsit** [comment]
@LizardWizardGB I hear you on that. I’ve been thinking about making the text part only as wide as the title part, so the thumb/voting area provides kind of a left margin.

**Comment by SineSwiper** [comment]
> A line of text shouldn't be no more than 90 characters long to keep readability, but now we are up at 140 characters.

Excuse me, but what standard are you quoting for that?  We ditched monochrome green-screen monitors with limited character widths in the 1970s.

This is [Reddit, your competitor, pulling off fullscreen mode](https://i.imgur.com/7bPiICN.png) just fine.  Even with Reddit's restrictive comment width inside the posts, they are still achieving about 110 characters max width.  Even [your own GitHub comment](https://i.imgur.com/Bt8fpFO.png) is 128 characters long, and is plenty readable.

Let's get rid of tiny, tiny paragraph widths, and get rid of ancient 1970s standards.

**Comment by jsit** [comment]
> > A line of text shouldn't be no more than 90 characters long to keep readability, but now we are up at 140 characters.
> 
> Excuse me, but what standard are you quoting for that?

This is a basic typographical truth that is universally understood.

**Comment by SineSwiper** [comment]
I think I just proved that highly-popular and battle-tested web software do not follow this imaginary 90-character limit.

**Pull request merged**